rubygem-foreman_deployments/rubygem-foreman_deployments.spec

```diff
@@ -0,0 +1,115 @@
+# This package contains macros that provide functionality relating to
+# Software Collections. These macros are not used in default
+# Fedora builds, and should not be blindly copied or enabled.
+# Specifically, the "scl" macro must not be defined in official Fedora
+# builds. For more information, see:
+# http://docs.fedoraproject.org/en-US/Fedora_Contributor_Documentation/1/html/Software_Collections_Guide/index.html
+
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_deployments
+
+%define rubyabi 1.9.1
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: A plugin adding Multi-Host Deployment support into the Foreman
+Group: Applications/System
+License: GPLv3
+URL: https://github.com/theforeman/foreman_deployments
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: foreman >= 1.8.0
+Requires: %{?scl_prefix_ruby}rubygems
+Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.7.3
+Requires: %{?scl_prefix}rubygem(foreman-tasks) < 0.8.0
+Requires: %{?scl_prefix}rubygem(safe_yaml) >= 1.0.0
```

Comment:
As per #885, please just make sure the gemspec gets updated too.  No need to do it and release it now for this to get merged, but just for the next release.  It's useful when they're in sync.